Assets/Scripts/Gameplay/Action/ActionUtils.cs

```diff
@@ -212,7 +213,10 @@ public static Vector3 GetDashDestination(Transform characterTransform, Vector3 t
 
 
     }
+}
 
+namespace Unity.Multiplayer.Samples.BossRoom.Actions
```

Comment:
why is this a separate namespace?

Reply:
Felt like it should be.

Reply:
we should at least be consistent with namespace prefix in the same file

Reply:
Oh I see, I didn't udnerstand the comment at first - you're right